### PR TITLE
Return non found page when a team doesn't exist

### PR DESCRIPTION
**Pull request by jpmaHub**
* Add test for pages teams

### CODE REVIEW

File: pages/teams/[id].tsx [hunk 1]
```diff
@@ -24,6 +25,7 @@ interface Props {
 
 const TeamPage = ({ users, team, forms }: Props): React.ReactElement => {
   return (
+    team ? (
```
Comment (acodeninja) [inline pages/teams/[id].tsx:28]: You can probably get rid of this ternary as the 404 is caught in the `getServerSideProps`.

Comment (jpmaHub) [reply pages/teams/[id].tsx:28]: removing the ternary causing this error `Unhandled Runtime Error
TypeError: Cannot read properties of undefined (reading 'map')` in page

Comment (acodeninja) [reply pages/teams/[id].tsx:28]: Interesting, lets try and fix that then...

**Review by acodeninja — APPROVED** [review]
LGTM 👍

**Pull request merged**